src/Tools/MicroBuild/run-gitlink.ps1

```diff
@@ -0,0 +1,41 @@
+# Run GitLink on the binaries that we are producing.
```

Comment:
Sorry, I should have added more detail to this commit about that.  I had put a write up of the problem in our previous convo and didn't think to add it here.  

The PR has a discussion of why this approach was taken now.